fenix/app/src/androidTest/java/org/mozilla/fenix/ui/SitePermissionsTest.kt

```diff
@@ -327,4 +328,26 @@ class SitePermissionsTest {
             }
         }
     }
+
+    @Test
+    fun verifyPermissionPromptForInsecureWebsiteTest() {
+        val insecureWebsite = "http://chrisdavidmills.github.io/location-finder-permissions-api"
+        val secureWebsite = "https://chrisdavidmills.github.io/location-finder-permissions-api"
```

Comment:
I would investigate more on how to not enforce HTTPS on our github testapp and host a page there. @AaronMT is there a way to disable enforcing HTTPS on https://github.com/mozilla-mobile/testapp ?

Reply:
Only if we had a custom domain.

![Screenshot 2023-07-18 at 10 53 00 AM](https://github.com/mozilla-mobile/firefox-android/assets/102331/e3a05845-4e86-4eb5-a0b4-b35f351014e8)

I'll look for an example of what desktop uses.